community/contributing/updating_the_class_reference.rst

```diff
@@ -1,151 +1,176 @@
+             <toc-class-ref>`, hosted in the `godot-docs
+             <https://github.com/godotengine/godot-docs>`_ repository.
 
-Get started with GitHub
------------------------
+Getting started with GitHub
+---------------------------
 
-If you're new to Git and GitHub, this guide will help you get started. You'll learn to:
+If you're new to using GitHub, the platform we use to develop Godot, this guide
+will help you get started. You will learn to:
 
 - Fork and clone Godot's repository
 - Keep your fork up to date with other contributors
-- Create a pull request so your improvements end in the official docs
+- Create a pull request to submit your improvements to the official docs
 
-.. note:: If you're new to Git, the version control system Godot uses, go through `GitHub's interactive guide <https://try.github.io/levels/1/challenges/1>`_. You'll learn some essential vocabulary and get a sense for the tool.
+.. note:: If you're new to Git, the version control system Godot uses, start
+          with `GitHub's interactive guide
+          <https://try.github.io/levels/1/challenges/1>`_. You'll learn some
+          essential vocabulary and get a sense for how the tool works.
 
-Fork Godot
-~~~~~~~~~~
+Forking Godot
+~~~~~~~~~~~~~
 
-Fork the Godot Engine into a GitHub repository of your own.
+Start by forking the Godot Engine into a GitHub repository of your own. Read the
+`GitHub forking guide <https://guides.github.com/activities/forking/>`_ to learn
+to create forks.
 
 Clone the repository on your computer:
 
 ::
 
     git clone https://github.com/your_name/godot.git
 
-Create a new branch to make your changes. It makes it a lot easier to sync your improvements with other docs writers. It's also easier to clean up your repository if you run into any issues with Git.
+Create a new branch to make your changes. It makes it a lot easier to
+synchronize your improvements with other contributors. It's also easier to clean
+up your repository if you run into any issues with Git.
 
 ::
 
     git checkout -b your-new-branch-name
 
-The new branch is the same as your master branch until you start to write API docs. In the ``doc/`` folder, you'll find the class reference.
+The new branch is the same as your master branch until you start to write API
+docs. In the ``doc/`` folder, you will find the class reference.
 
-How to keep your local clone up-to-date
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Keeping your local clone up-to-date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Other writers contribute to Godot's documentation. Your local repository will fall behind it, and you'll have to synchronize it. Especially if other contributors update the class reference while you work on it.
+Other writers contribute to Godot's documentation. Your local repository will
+fall behind it. You will have to synchronize it, especially if other
+contributors update the class reference while working on it.
```

Comment:
The change to "while working on it" doesn't make things clearer IMO, as you don't know who is the subject. Before it was about while *you* are working on it, now I'd understand it as "contributors can update the class reference while they work on the class reference" (obvious :)).